modules/WES/domains/level_1.yaml

```diff
@@ -0,0 +1,179 @@
+name: Level1
+id: https://w3id.org/htan/wes/level_1
+description: Bulk Whole Exome Sequencing Level 1 - Raw files
+imports:
+  - linkml:types
+
+prefixes:
+  htan: https://w3id.org/htan/
+  linkml: https://w3id.org/linkml/
+  NCIT: http://purl.obolibrary.org/obo/NCIT_
+
+default_prefix: htan
+
+enums:
+  LibraryLayoutEnum:
+    permissible_values:
+      "Paired-end":
+        description: Paired-end sequencing
+      "Single-end":
+        description: Single-end sequencing
+
+  LibrarySelectionMethodEnum:
+    permissible_values:
+      "Hybrid Selection":
+        description: Hybrid selection method
+      "PCR":
+        description: PCR-based selection
+      "RANDOM":
+        description: Random selection
+      "other":
+        description: Other selection method
+
+  SequencingPlatformEnum:
+    permissible_values:
+      "ILLUMINA":
+        description: Illumina sequencing platform
+      "PACBIO_SMRT":
+        description: PacBio SMRT sequencing platform
+      "OXFORD_NANOPORE":
+        description: Oxford Nanopore sequencing platform
+      "ION_TORRENT":
+        description: Ion Torrent sequencing platform
+      "COMPLETE_GENOMICS":
+        description: Complete Genomics sequencing platform
+      "CAPILLARY":
+        description: Capillary sequencing platform
+      "LS454":
+        description: 454 sequencing platform
+      "ABI_SOLID":
+        description: ABI SOLID sequencing platform
+      "HELICOS":
+        description: Helicos sequencing platform
+      "BGISEQ":
+        description: BGI sequencing platform
+
+classes:
+  BulkWESLevel1:
+    description: Bulk Whole Exome Sequencing Level 1 - Raw files
+    attributes:
+      COMPONENT:
+        range: string
+        required: true
+        description: Category of metadata (should be "Bulk WES Level 1")
+      FILENAME:
+        range: string
+        required: true
+        description: Name of the file
+      FILE_FORMAT:
+        range: string
+        required: true
+        description: Format of the file (e.g., fastq, bam)
+      HTAN_PARENT_BIOSPECIMEN_ID:
+        range: string
+        required: true
+        description: HTAN Biospecimen ID of the parent biospecimen
+      HTAN_DATA_FILE_ID:
```

Comment:
I think we need to implement our HTAN 'core' attributes model as part of this PR (or a sperate one that gets merged into wes first and then into main)

Reply:
We also want to avoid defining these attributes in every file-based module